var/spack/repos/builtin/packages/charmpp/package.py

```diff
@@ -187,9 +187,7 @@ def charmarch(self):
             ("linux", "ppc", "pami"): "pami-linux-ppc64le",
             ("linux", "ppc", "verbs"): "verbs-linux-ppc64le",
             ("linux", "arm", "netlrts"): "netlrts-linux-arm7",
-            ("linux", "arm", "multicore"): "multicore-arm7",
             ("linux", "aarch64", "netlrts"): "netlrts-linux-arm8",
-            ("linux", "aarch64", "multicore"): "multicore-arm8",
```

Comment:
Can you move the "old" versions to the < 6.11.0 version below?

Reply:
Yes, sorry, I did not realise this was the intended change.